src/test/java/com/iexec/worker/tee/scone/LasServiceTests.java

```diff
@@ -149,9 +149,9 @@ void shouldNotStartLasServiceSinceClientError() throws Exception {
     }
 
     @Test
-    void shouldNotStartLasServiceSinceClientException() throws Exception {
+    void shouldNotStartLasServiceSinceClientException() {
         when(dockerService.getClient(REGISTRY_NAME, REGISTRY_USERNAME, REGISTRY_PASSWORD))
-                .thenThrow(Exception.class);
+                .thenThrow(RuntimeException.class);
```

Comment:
Throwing an `IllegalArgumentException` would make sense, don't you think?

Reply:
comment added with the reason behind this necessity